arm_robots/scripts/trajectory_follower_node.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```

Comment:
I am not happy about having to specify `python3`, however `rostest` invokes `python2` if the shebang is just for `python`.

Specifying `python3` is not currently an execution problem in our case, since this code only supports `python3`. However deviating from the default `/usr/bin/env python` adds more mental overhead in the future

Reply:
yea specifying 3 is fine :shrug: